ArduPlane/Parameters.h

```diff
@@ -93,14 +93,14 @@ class Parameters {
         k_param_scheduler,
         k_param_relay,
         k_param_takeoff_throttle_delay,
-        k_param_skip_gyro_cal, // unused
+        k_param_takeoff_rudder_mix,
```

Comment:
@OXINARF yeah agreed, I mentioned this issue in my last comment as well(the last statement). Anyways, I will move it down to G2, as I don't see any way around this problem.